docs/extend/modular-resources/create-type.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Define a New Resource Type"
+linkTitle: "New API Type"
+weight: 30
+type: "docs"
+tags: ["rdk", "extending viam", "modular resources", "API"]
+description: "Define an API for a new type of resource."
+no_list: true
+---
+
+You need to define a new {{% glossary_tooltip term_id="resource" text="resource" %}} *type* (and an API for that type) if:
+
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) types.
+- You have a resource that could fit into an existing type, for example, [base](/components/base/), but you want to define an API with different endpoints than the ones in the standard Viam [API](/program/apis/) for that type.
```

Comment:
I think a more explicit example would be really helpful here

Reply:
```suggestion
- You have a resource that could fit into an existing type, but you want to define an API with different endpoints than the ones in the standard Viam [API](/program/apis/) for that type.
For example, you have a [sensor](/components/sensor/) and you want to create a `Calibrate` endpoint.
```